scripts/get_openstack_plaintext_docs.sh

```diff
@@ -68,6 +73,8 @@ for project in "${OS_PROJECTS[@]}"; do
     cd -
 done
 
-# TODO(lucasagomes): Should we delete the working directory ?!
+rm -rf $CURR_DIR/openstack-docs-plaintext
 cp -r $WORKING_DIR/openstack-docs-plaintext $CURR_DIR
+
+# TODO(lucasagomes): Should we delete the working directory ?!
```

Comment:
https://github.com/road-core/rag-content/blob/ae829e6d43860ad00cee7294b781f6ff7f82a5ac/scripts/get_ocp_plaintext_docs.sh#L6

Reply:
Yeah, sometimes I find useful to not delete it. I think I will leave it out until we get the script to it's final version